runtime-sdk/src/modules/accounts/mod.rs

```diff
@@ -43,6 +45,14 @@ pub enum Error {
     #[error("forbidden by policy")]
     #[sdk_error(code = 3)]
     Forbidden,
+
+    #[error("core: {0}")]
+    #[sdk_error(transparent)]
+    Core(
+        #[from]
+        #[source]
```

Comment:
Since `from` implies `source` you only need `from`.

Reply:
thx I'll try that

Reply:
looks good